src/strife/d_main.c

```diff
@@ -1018,7 +1018,7 @@ void PrintDehackedBanners(void)
     }
 }
 
-static struct 
```

Comment:
Same here, this has nothing to do with this PR.

Reply:
As in the case of Doom, I removed the commit in question.